### ISSUE TITLE
Suggest making this less visible to respect the author's wishes?

### ISSUE THREAD

**Issue by chris838** [domenic/worm-ebook-scraper#12]
This repo is the number one result for "worm ebook". The author has explicitly said he doesn't mind people making their own ebooks but that publicising such methods damages his ability to pursue traditional ebook publishing routes.

Not saying this should be taken down (it's super useful!) but can we at least make it less conspicuous? This guy spends 50+ hours per week writing and has a following of millions, I just wish there was a way he could profit more from his awesome work.

**Comment by leonxu1** [comment]
Agreed with chris, Wildbow has said that publicly available scrapers mess with his ability to talk to publishers ([source](https://www.reddit.com/r/Parahumans/comments/6cusa0/wildbow_ebook_scraper_question/dhxqakv/)), and have scared away at least one interested publisher in the past.  On the one hand, you've got a super useful tool here, but on the other, you're kinda hurting and disrespecting the author. At the very very least, you could credit him directly in the README, maybe link to his Patreon, and talk to him directly about whether he wants this taken down.

**Comment by Retsam** [comment]
Yeah, the [FAQ for Ward (Worm 2)](https://www.parahumans.net/f-a-q/) specifically asks that scrapers be publicly available:

> Q4: Ebook?  Can I have a copy?  Is there going to be one?  What about Worm?
> 
> A4: Ebooks and the like are just about inevitable – Wildbow is in talks with an agent and has talked to some major publishers.  The current plan is to get the editing done and shop it around – with the established fanbase being what it is, Wildbow has a lot of freedom.  He’d really like to do stuff like keep the website up and keep the integrity of his work, and the fact that the works are getting reads, money, reviews, fanart, and such really help with negotiating that.
> 
> Unfortunately, dissemination of ebooks sort of muddies the waters and hampers those negotiations, whether it’s Wildbow doing it or others.  For the time being, ebooks are not being distributed, and **Wildbow would ask that people don’t set up sites, share scrapers or otherwise mass-distribute them**.  There’s just too many questions of rights and ownership (who is the actual creator/distributor?  Who was first?), whether the author really has control over their own property, and if the publisher feels it is worthwhile to get the rights to distribute the ebook if it’s already being put out there.
> 
> There’s lots of really easy and obvious answers to all of these points, but in reality, publishers are skittish.  Wildbow has already missed out on one opportunity to get Worm out there because the other party got scared off by the mass-distribution via. one scraper.  If you guys see such a site or person out in the wild, maybe send them a message, point them here/copy paste all this, or ask them politely to quit it, and if they don’t, let Wildbow know so he can figure it out.

---

So, yeah, Wildbow would definitely prefer that this public github repository not exist.  It's an awesome tool (I was really glad to have an ebook version, personally), and the FAQ does go on to say:

> Q4b: Ebooks… what if I made one for personal use, or to share with a best friend or family member?
> 
> A4b: If it’s not obvious to the publishers and not known to Wildbow, then no problem arises.

So perhaps this repo could be taken down (or made private), and the script could be distributed more discreetly to people who want it?  (e.g. via the parahumans subreddit)

**Comment by makeworld-the-better-one** [comment]
At the risk of being controversial, this repository has been archived [here](https://archive.org/details/github.com-domenic-worm-scraper_-_2019-08-17_03-39-59).

**Comment by GledsonAfonso** [comment]
Doesn't make any sense in making it **less** visible. People, this is Github. How many people do you think are able to use this **and** read Wildbow works?

**Comment by Retsam** [comment]
You don't need to know how to open a PR to be able to google "worm ebook" click the (currently) third result, scroll down a little, read the instructions in a readme and follow them.  (I imagine most people who are reading million word web serials *do* have basic reading comprehension.)

**Comment by GledsonAfonso** [comment]
> You don't need to know how to open a PR to be able to google "worm ebook" click the (currently) third result, scroll down a little, read the instructions in a readme and follow them. (I imagine most people who are reading million word web serials _do_ have basic reading comprehension.)

Basic reading comprehension? Sure. Capable of installing node and using an CLI? Not so sure.

**Comment by Retsam** [comment]
The README has full instructions for installing node (click link, click current, run the OS installer), how to open a terminal in your OS, and exactly what commands to run. 

But all of that is really a moot point.  The whole point is that the widespread availability has scared off publishers and hurt the author.  I don't think a potential publisher is going to care at all that scraping the ebook requires some minimal technical skill.  

And more so, it really doesn't *matter* what I think, or what you think about potential publishers.  The author has made their wishes on the matter clear, and I think it's just decency to respect their wishes as much as possible.

**Comment by GledsonAfonso** [comment]
> The README has full instructions for installing node (click link, click current, run the OS installer), how to open a terminal in your OS, and exactly what commands to run.
>

This really does not mean anything from the perspective of someone that probably doesn't understand development in any way (even if this seen trivial for us).

> But all of that is really a moot point. The whole point is that the widespread availability has scared off publishers and hurt the author. I don't think a potential publisher is going to care at all that scraping the ebook requires some minimal technical skill.
> 
> And more so, it really doesn't _matter_ what I think, or what you think about potential publishers. The author has made their wishes on the matter clear, and I think it's just decency to respect their wishes as much as possible.
>

I do see your point and I do agree none of this matter. Just like this issue and this discution. I will keep using it nonetheless, just like I will keep it forked for other people to use it. It is a really good peace of software and it does not deserve to be taken down just because people like you see a morality problem in it.

**Comment by Retsam** [comment]
Apparently "you should respect the wishes of an author who writes a free story that you enjoy to the best of your ability" is a controversial morality statement.  TIL.

And apparently what the "software deserves" outranks what the author who spent thousands of hours writing this free story deserves.  Shame.

**Comment by GledsonAfonso** [comment]
> Apparently "you should respect the wishes of an author who writes a free story that you enjoy to the best of your ability" is a controversial morality statement. TIL.
> 

Didn't say it was controversial, but it is a morality statement.

> And apparently what the "software deserves" outranks what the author who spent thousands of hours writing this free story deserves. Shame.
>

Didn't say that too. They are two different cases and I just state that, in the case of the software, it didn't deserve to be taken down and, in reality, it was a silly solution for the problem that the author is having.

But okay, this is just pointless anyway. Have a good one!
